roles/comanage/defaults/main.yml

```diff
@@ -2,8 +2,8 @@
 http_proto: https
 
 comanage_repo_url: "https://github.com/Internet2/comanage-registry"
-comanage_repo_version: "master"
-comanage_install_dir: "/opt/comanage/"
+comanage_repo_version: "tree/3.1.0"
+comanage_install_dir: "/opt/comanage"
 comanage_path: "{{ comanage_install_dir }}/comanage"
```

Comment:
Kun je deze in een aparte pull request zetten?  Ik wil namelijk de fixes voor de plugins wel in de volgende release, maar de upgrade naar comanage 3.1 nog niet.

Verder lijkt het me beter om de versie van comanage hier bij default op master te laten staan, en hem in de environment/vm en environment/surf group_vars te fixen op een specifieke versie.

Reply:
Die master zet in in separate. De remove van trailing slash wou ik nu wel doorzetten, hij resulteert in dubbele slashes in latere tasks.